config/routes.rb

```diff
@@ -1,3 +1,7 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  get 'welcome/index'
+
+  resources :neighborhoods
```

Comment:
Using `resources :neighborhoods` will give you seven different routes. Since you are only working on the index so far, you could write `resources :neighborhoods, only: [:index]` to limit it to just that one route.

Reply:
I thought it would be easier to get the scaffolding up by following the suggestions in the Rails tutorial, but you're right that I only need index presently.  (I did cheat a little and delete the coffee and sass files that I knew I wouldn't need so I'm not opposed to removing some of the extra routes.) . If I run the code above, will that overwrite the current resources file, or is that something I'd use the next time I create a new resource?  When starting a new project, what do you usually do?

Reply:
I think following the tutorial makes the most sense for now. I mentioned the `only` option mostly just so you would know about it for the future.

> If I run the code above, will that overwrite the current resources file
I'm not sure I know what you mean. To use the `only` option you would only need to change line 4 of your `config/routes.rb`. Nothing else would change.